src/istio/control/http/attributes_builder.cc

```diff
@@ -169,6 +169,11 @@ void AttributesBuilder::ExtractCheckAttributes(CheckData *check_data) {
     }
   }
   builder.AddString(utils::AttributeName::kContextProtocol, protocol);
+
+  builder.AddStringMap(utils::AttributeName::kRequestQueries,
```

Comment:
Can you move these two to ExtractRequestHeaderAttributes?

Reply:
Done.